docs/bot/synapse_module.md

```diff
@@ -33,27 +37,27 @@ modules:
       # to already be joined to the room - Mjolnir will not automatically join
       # these rooms.
       ban_lists:
-         - "!roomid:example.org"
+        - "!roomid:example.org"
       #message_max_length:
-         # Limit the characters in a message (event body) that a client can send in an event on this server.
-         # By default there is no limit (beyond the the limit the spec enforces on event size).
-         # Uncomment if you want messages to be limited to 510 characters.
-         #threshold: 510
+      # Limit the characters in a message (event body) that a client can send in an event on this server.
+      # By default there is no limit (beyond the the limit the spec enforces on event size).
+      # Uncomment if you want messages to be limited to 510 characters.
+      #threshold: 510
 
-         # Limit messages only in certain rooms rooms.
-         # By default all rooms will enforce the limit.
-         # Uncomment if you want messages to only be subject to character limits in certain rooms.
-         #rooms:
-         #  - "!vMvyOCeCxHsggkmALd:localhost:9999"
+      # Limit messages only in certain rooms rooms.
+      # By default all rooms will enforce the limit.
+      # Uncomment if you want messages to only be subject to character limits in certain rooms.
+      #rooms:
+      #  - "!vMvyOCeCxHsggkmALd:localhost:9999"
 
-         # Also hide messages from remote servers that are over the `message_limit`.
-         # By default only events from this server will be limited.
-         # WARNING: Remote users on other servers will still be able to messages over the limit.
-         # Uncomment to enforce the `message_limit` on events from remote servers.
-         #remote_servers: true
+      # Also hide messages from remote servers that are over the `message_limit`.
+      # By default only events from this server will be limited.
+      # WARNING: Remote users on other servers will still be able to messages over the limit.
+      # Uncomment to enforce the `message_limit` on events from remote servers.
+      #remote_servers: true
```

Comment:
Nope. This is wrong. Editor is being dumb when its removing indentations here because they have to be here for this to be a valid template to use.

Reply:
that's incorrect, this is all valid.

Reply:
![image](https://github.com/the-draupnir-project/draupnir-documentation/assets/50846879/ba01e134-529a-4aca-be69-855c17c02ef9)